### PR TITLE
Resolve some "NEXT_MAJOR" comments

### PR DESCRIPTION
**Pull request by phansys**
## Subject

Resolve some "NEXT_MAJOR" comments.

- [x] Add an upgrade note;
- [ ] Fix the tests.

### CODE REVIEW

**Comment by phansys** [comment]
I need help with the tests at `GetShortObjectDescriptionActionTest`, since `TemplateRegistry::getTemplate()` is returning `null`:
```
Twig\Error\LoaderError: Template "" is not defined.
```
Thank you in advance.

File: src/Admin/AdminInterface.php [hunk 1]
```diff
@@ -114,13 +114,6 @@ public function setTemplates(array $templates);
      */
     public function setTemplate($name, $template);
```
Comment (core23) [inline src/Admin/AdminInterface.php]: You should remove this method too
